Makefile

```
@@ -21,3 +67,25 @@ build_image_onnx:
 	docker push $(IMAGE_ONNX):$(IMAGE_ONNX_TAG)
+
+run_local_image_onnx:
+	docker run --rm \
+		-v ~/.config/gcloud/application_default_credentials.json:/root/.config/gcloud/application_default_credentials.json:ro \
+		-e GOOGLE_APPLICATION_CREDENTIALS=/root/.config/gcloud/application_default_credentials.json \
+		-e GOOGLE_CLOUD_PROJECT=$(GOOGLE_CLOUD_PROJECT) \
+		-e AIP_STORAGE_URI=$(AIP_STORAGE_URI) \
+		-e MODEL_NAME=$(MODEL_NAME) \
+		-p 8080:8080 $(IMAGE_ONNX):$(IMAGE_ONNX_TAG)_local
+
+test_image_onnx:
```

checkmake (0.3.2)

[warning] Target body for "test_image_onnx" exceeds allowed length of 5 lines (11).

(maxbodylength)